pkg/actuators/machine/vm.go

```diff
@@ -36,21 +36,33 @@ func createVM(mscp *machineScope, userData []byte) (*nutanixClientV3.VMIntentRes
 		userdataEncoded := base64.StdEncoding.EncodeToString(userData)
 
 		// Create the VM
-		klog.Infof("To create VM with name %s, and providerSpec: %+v", vmName, *mscp.providerSpec)
+		klog.Infof("To create VM with name %q, and providerSpec: %+v", vmName, *mscp.providerSpec)
```

Comment:
Logging the whole providerspec seems excessive, are there portions of it we may want to log rather than the whole thing? Perhaps increasing the log level here would be appropriate if you want this as a debug option?

Reply:
Changed the log level to V(5) for this log